castervoice/apps/griddouglas.py

```diff
@@ -63,4 +121,4 @@ class GridControlRule(MergeRule):
     rule = GridControlRule(name="Douglas")
     gfilter.run_on(rule)
     grammar.add_rule(rule)
-    grammar.load()
+    grammar.load()
```

Comment:
Does no newline matter for python files?

Reply:
Won't break anything but it should be added